.azure/ipu-tests.yml

```
@@ -77,6 +77,12 @@ jobs:
           python ./requirements/pytorch/adjust-versions.py $fpath; \
         done
         pip install -e .[dev]
+
+        # TODO: Remove this uninstall command
+        # https://github.com/microsoft/DeepSpeed/commit/b587c7e85470329ac25df7c7c2521ff9b2833db7
+        # gets released
+        pip uninstall deepspeed
```

I suggest doing this instead, so that deepspeed is not even installed

```suggestion
        pip install -e .[extra,examples]
```